.travis.yml

```diff
@@ -22,15 +31,29 @@ matrix:
       env: DB=MYSQL CORE_RELEASE=3.1
     - php: 5.6
       env: DB=PGSQL CORE_RELEASE=3.2
-  allow_failures:
-    - php: 7.0
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3.3
+    # Allowed to fail, meaning delayed code coverage reports
+    - php: 5.5.9
+      env: DB=MYSQL CORE_RELEASE=3.2 COVERAGE=1
 
 before_script:
+  - phpenv rehash
   - composer self-update || true
   - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
   - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
   - cd ~/builds/ss
-  - composer install
 
 script:
-  - vendor/bin/phpunit iframe/tests
+  # Execute tests with no coverage.  This is the fastest option
+  - "if [ \"$COVERAGE\" = \"0\" ]; then vendor/bin/phpunit $MODULE_PATH/tests/; fi"
+
+  # Execute tests with coverage.  Do this for a small
+  - "if [ \"$COVERAGE\" = \"1\" ]; then vendor/bin/phpunit --coverage-clover=coverage.clover $MODULE_PATH/tests/; fi"
+
+after_script:
+  - "if [ \"$COVERAGE\" = \"1\" ]; then mv coverage.clover ~/build/$TRAVIS_REPO_SLUG/; fi"
```

Comment:
See https://docs.travis-ci.com/user/environment-variables#Defining-Variables-in-.travis.yml, 

The slug (in form: `owner_name/repo_name`) of the repository currently being built. (for example, “travis-ci/travis-build”).